packages/mobile/src/components/user-link/UserLink.tsx

```diff
@@ -63,6 +65,10 @@ export const UserLink = (props: UserLinkProps) => {
           animatedPressed.value = withTiming(0, motion.press)
         }
       }}
+      onPress={() => {
```

Comment:
where is the other press defined? in the user-link itself i guess? is it chill to have two competing pressables?

Reply:
yeah inside the text link. doesn't seem to cause issues to have both